src/geos.h

```diff
@@ -4,7 +4,9 @@
 #include <Python.h>
 
 /* To avoid accidental use of non reentrant GEOS API. */
+#ifndef GEOS_USE_ONLY_R_API
```

Comment:
This change was required by adding `
# distutils: define_macros=GEOS_USE_ONLY_R_API` to the Cython modules; we were getting warnings about redeclaring the macro.